### ISSUE TITLE
Treat Finder aliases the same as symlinks (i.e follow them)

### ISSUE THREAD

**Issue by danielbayley** [atom/tree-view#529]
As it stands, Atom opens the actual Finder alias when selected in Tree view, resulting in binary gibberish. Maybe there's a way to follow the link as it would a symlink?

OS X 10.10.4
Atom 1.0.2

**Comment by petonic** [comment]
I agree with this enhancement request.   TextMate's file browser does use Finder Aliases correctly.  It would be great to have this for those of us who have just recently switched over.

**Comment by 50Wliu** [comment]
Can someone run `fs.statSync` on an alias and report back what it prints out?

**Comment by rsese** [comment]
I think I did it right @50Wliu, let me know if you need anything else/something else:

```json
{"dev":16777220,"mode":33188,"nlink":1,"uid":501,"gid":20,"rdev":0,"blksize":4096,"ino":72994372,"size":896,"blocks":8,"atimeMs":1547089098000,"mtimeMs":1547087093000,"ctimeMs":1547087093000,"birthtimeMs":1547087093000,"atime":"2019-01-10T02:58:18.000Z","mtime":"2019-01-10T02:24:53.000Z","ctime":"2019-01-10T02:24:53.000Z","birthtime":"2019-01-10T02:24:53.000Z"}
```

**Comment by thomasmalley** [comment]
Running into the same Problem. Can I provide some Help to fix this?

**Comment by 50Wliu** [comment]
So it looks like there's no documentation for how Finder aliases are represented, and the only Node API I can find (https://github.com/LinusU/node-alias) seems like it relies on anecdotal evidence and reverse-engineering. Unfortunately, I don't think this will be fixed anytime soon unless good library support is added for it.

**Comment by danielbayley** [comment]
@50Wliu Well how does TextMate's file browser handle it?

**Comment by 50Wliu** [comment]
I'm assuming that they use Mac's native APIs to do so.
